tst/EnergyPlus/unit/ZoneContaminantPredictorCorrector.unit.cc

```diff
@@ -390,3 +390,220 @@ TEST_F(EnergyPlusFixture, ZoneContaminantPredictorCorrector_CorrectZoneContamina
+    DataContaminantBalance::ZoneCO2SetPoint(2) = 500.0;
+    DataContaminantBalance::ZoneCO2SetPoint(3) = 550.0;
+    DataContaminantBalance::ZoneAirCO2(1) = DataContaminantBalance::ZoneCO21(1);
+    DataContaminantBalance::ZoneAirCO2(2) = DataContaminantBalance::ZoneCO21(2);
+    DataContaminantBalance::ZoneAirCO2(3) = DataContaminantBalance::ZoneCO21(3);
+
+    Real64 PriorTimeStep;
+
+    TimeStepSys = 15.0 / 60.0; // System timestep in hours
+    PriorTimeStep = TimeStepSys;
+
+    ZoneEquipConfig.allocate(3);
+    ZoneEquipConfig(1).ZoneName = "Zone 1";
+    ZoneEquipConfig(1).ActualZoneNum = 1;
+
+    ZoneEquipConfig(1).NumInletNodes = 2;
+    ZoneEquipConfig(1).InletNode.allocate(2);
+    ZoneEquipConfig(1).InletNode(1) = 1;
+    ZoneEquipConfig(1).InletNode(2) = 2;
+    ZoneEquipConfig(1).NumExhaustNodes = 1;
+    ZoneEquipConfig(1).ExhaustNode.allocate(1);
+    ZoneEquipConfig(1).ExhaustNode(1) = 3;
+    ZoneEquipConfig(1).NumReturnNodes = 1;
+    ZoneEquipConfig(1).ReturnNode.allocate(1);
+    ZoneEquipConfig(1).ReturnNode(1) = 4;
+
+    ZoneEquipConfig(2).ZoneName = "Zone 2";
+    ZoneEquipConfig(2).ActualZoneNum = 2;
+
+    ZoneEquipConfig(2).NumInletNodes = 1;
+    ZoneEquipConfig(2).InletNode.allocate(1);
+    ZoneEquipConfig(2).InletNode(1) = 6;
+    ZoneEquipConfig(2).NumExhaustNodes = 0;
+    ZoneEquipConfig(2).NumReturnNodes = 1;
+    ZoneEquipConfig(2).ReturnNode.allocate(1);
+    ZoneEquipConfig(2).ReturnNode(1) = 7;
+
+    ZoneEquipConfig(3).ZoneName = "Zone 3";
+    ZoneEquipConfig(3).ActualZoneNum = 3;
+
+    ZoneEquipConfig(3).NumInletNodes = 1;
+    ZoneEquipConfig(3).InletNode.allocate(1);
+    ZoneEquipConfig(3).InletNode(1) = 8;
+    ZoneEquipConfig(3).NumExhaustNodes = 0;
+    ZoneEquipConfig(3).NumReturnNodes = 1;
+    ZoneEquipConfig(3).ReturnNode.allocate(1);
+    ZoneEquipConfig(3).ReturnNode(1) = 9;
+
+    Node.allocate(10);
+
+    Zone.allocate(3);
+    Zone(1).Name = ZoneEquipConfig(1).ZoneName;
+    Zone(1).ZoneEqNum = 1;
+    ZoneEqSizing.allocate(3);
+    CurZoneEqNum = 1;
+    Zone(1).Multiplier = 1.0;
+    Zone(1).Volume = 1000.0;
+    Zone(1).SystemZoneNodeNumber = 5;
+    Zone(1).ZoneVolCapMultpMoist = 1.0;
+    Zone(2).Name = ZoneEquipConfig(2).ZoneName;
+    Zone(2).ZoneEqNum = 1;
+    Zone(2).Multiplier = 1.0;
+    Zone(2).Volume = 1000.0;
+    Zone(2).SystemZoneNodeNumber = 5;
+    Zone(2).ZoneVolCapMultpMoist = 1.0;
+    Zone(3).Name = ZoneEquipConfig(3).ZoneName;
+    Zone(3).ZoneEqNum = 1;
+    Zone(3).Multiplier = 1.0;
+    Zone(3).Volume = 1000.0;
+    Zone(3).SystemZoneNodeNumber = 5;
+    Zone(3).ZoneVolCapMultpMoist = 1.0;
+
+    OutBaroPress = 101325.0;
+
+    NumZoneReturnPlenums = 0;
+    NumZoneSupplyPlenums = 0;
+
+    OAMFL.allocate(3);
+    VAMFL.allocate(3);
+    EAMFL.allocate(3);
+    CTMFL.allocate(3);
+    MDotOA.allocate(3);
+    MDotOA = 0.001;
+    ScheduleManager::Schedule.allocate(1);
+
+    ScheduleManager::Schedule(1).CurrentValue = 1.0;
```

Comment:
It's nice that this is so easy, I had assumed overriding the schedule would be a bit more effort.